Description: Install unstripped binaries, dh_strip takes care of this.
Author: Andreas Beckmann <debian@abeckmann.de>

--- a/utils.mk
+++ b/utils.mk
@@ -51,6 +51,7 @@
   STRIP_CMD           ?= true
   CFLAGS              += -O0 -g
 else
+  STRIP_CMD           ?= true # let dh_strip do the stripping
   STRIP_CMD           ?= strip
 endif
 
